### DISCUSSION TITLE
Running two desktop sessions at the same time (with different users)

### DISCUSSION THREAD

**Discussion by mrorgmode** [i3/i3#5926] [Q&A] [answered]
I would like to be logged in to two accounts at the same time, and be able to switch between the two desktop sessions:

 1. From GDM, login as user0 (i3-gnome), and start working.
 2. Somehow "escape" back to GDM and login as user1 (i3-gnome), and start working
 3. Somehow switch back to user0's session
 4. switch again...
 
 I suspect there is no function in i3 to "escape" back to GDM, but I thought I'd ask the community anyway.

**Comment by orestisfl** [comment, marked as answer]
Generally, you can switch to new ttys with Ctrl+Alt+Fx (x=1,2,3...) and then start a new i3 session there.

**Reply by mrorgmode** [reply]
Thousand thanks, and you'd better believe I mean it! That did exactly what I wanted! This is really helpful for me!  🙏🤝🙌
